styleguide/derek/_partials/_rug.ejs

```diff
@@ -3,28 +3,28 @@
 <div class="container rug">
   <div class="row">
     <div class="col-sm-3">
-      <h5 class="no-margin white">We've got your back</h5>
-      <h4 class="rug-subhead no-margin white">24<small>x</small>7<small>x</small>365</h4>
+      <h5 class="no-margin rug-head">We've got your back</h5>
+      <h4 class="no-margin rug-subhead">24<small>x</small>7<small>x</small>365</h4>
     </div>
     <div class="col-sm-3">
       <div class="row">
         <div class="col-md-2">
-          <i class="fa fa-mobile fa-4x white"></i>
+          <i class="fa fa-mobile fa-4x rug-icon"></i>
         </div>
         <div class="col-md-10 no-padding rug-contact-link rug-phone-numbers">
-          <span class="white"><a class="white rug-contact-link" href="tel:1-844-858-5963">1-844-858-5963</a> Sales</span><br/>
-          <span class="white"><a class="white rug-contact-link" href="tel:+18009614454">1-800-961-4454</a> Support</span>
+          <span class="rug-link"><a class="rug-link rug-link-hover" href="tel:1-844-858-5963">1-844-858-5963</a> Sales</span><br/>
```

Comment:
Not sure if it makes sense to use the same class for the `<span>` and for the `<a>` tag here. Perhaps the span should be `rug-text` and the `a` should stay as `rug-link`.